internal/pkg/cli/helpmsg/msg/app.txt

```diff
@@ -0,0 +1,12 @@
+
+Keboola CLI is a set of commands
```

Comment:
Trocha som vylepsoval texty, ked si to v dokumentacii, tak dobre napisal :)
Na konci sprintu chcem exportovat vsetky texty z funkcionalnych testov a poslat Hanke na review, takze texty staci asi iba rychlo prejst.